OpenRA.Game/PlayerDatabase.cs

```diff
@@ -0,0 +1,87 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Net;
+using OpenRA.Graphics;
+
+namespace OpenRA
+{
+	public class PlayerDatabase : IGlobalModData
+	{
+		public readonly string Profile = "https://forum.openra.net/openra/info/";
```

Comment:
This should be configurable through command line arguments, through the mod yaml or both.

Reply:
IGlobalModData classes can be overriden in mod.yaml.  See the PR description for syntax.

Reply:
Ok, I completely forgot about that during the review. Being able to override it through command line arguments would be a bonus, but the advantages of it are a bit questionable.